### ISSUE TITLE
In FilesystemManager.php line 126: Driver [] is not supported.

### ISSUE THREAD

**Issue by jayeshcspl** [arubacao/asset-cdn#8]
When I run php artisan asset-cdn:sync get following error.

In FilesystemManager.php line 126: Driver [] is not supported.

I have set following code in /config/filesystems.php

```
'default' => env('FILESYSTEM_DRIVER', 'local'),

    'cloud' => env('FILESYSTEM_CLOUD', 's3'),

    'disks' => [

        'local' => [
            'driver' => 'local',
            'root' => storage_path('app'),
        ],

        'public' => [
            'driver' => 'local',
            'root' => storage_path('app/public'),
            'url' => env('APP_URL').'/storage',
            'visibility' => 'public',
        ],

        's3' => [
            'driver' => 's3',
            'key' => env('XXXXXXXX'),
            'secret' => env('XXXXXXXXXXXXXXX'),
            'region' => env('ap-southeast-1'),
            'bucket' => env('bpn.svr.files.public'),
        ],

    ],
```

and in flie /config/asset-cdn.php

```
'use_cdn' => env('USE_CDN', true),

    'cdn_url' => 'https://s3.console.aws.amazon.com/s3/buckets/bpn.svr.files.public',

    'filesystem' => [
        'disk' => 'asset-cdn',

        'options' => [
            //
        ],
    ],

    'files' => [
        'ignoreDotFiles' => true,

        'ignoreVCS' => true,

        'include' => [
            'paths' => [
                //
            ],
            'files' => [
                //
            ],
            'extensions' => [
                //
            ],
            'patterns' => [
                //
            ],
        ],

        'exclude' => [
            'paths' => [
                //
            ],
            'files' => [
                //
            ],
            'extensions' => [
                //
            ],
            'patterns' => [
                //
            ],
        ],
    ],
```

![asset-cdn_sync_error](https://user-images.githubusercontent.com/46957760/85515401-b462cc00-b61a-11ea-888c-1f1840e1aa6d.png)

![bitbucket](https://user-images.githubusercontent.com/46957760/85517173-a31abf00-b61c-11ea-9bdc-edaa94fafdff.png)

**Comment by jayeshcspl** [comment]
Just notice that Method asset_cdn does not exist. (View:resources/views/admin/layouts/head.blade.php)

Anybody let me know what i miss?

![error](https://user-images.githubusercontent.com/46957760/85525674-eaa64880-b626-11ea-8f42-67c58f459278.png)

**Comment by arubacao** [comment]
What version of Laravel are you using?

**Comment by jayeshcspl** [comment]
I am using Laravel Framework 5.5.44

**Comment by arubacao** [comment]
My best guess is: you need to update your config `asset-cdn` --> `s3` in your `/config/asset-cdn.php` since you do not have a `asset-cdn` disk configured in `filesystems.php`

**Comment by jayeshcspl** [comment]
Thank you it sloved above issue.

Getting another issue. 
`Refused to apply style from 'https://s3.console.aws.amazon.com/s3/buckets/bpn.svr.files.public/assets/css/custom.css?region=ap-south-1' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.`

**Comment by arubacao** [comment]
"https://s3.console.aws.amazon.com/s3/buckets/bpn.svr.files.public/assets/css/custom.css?region=ap-south-1" cannot be accessed.
You need to make your bucket public

**Comment by jayeshcspl** [comment]
Already make it public

**Comment by arubacao** [comment]
Try to access the linked file with an incognito browser. Your file is not accessible through the general internet. Please read the documentation for bucket policies on AWS. You can have private files in public buckets.
You can also add 
```

**Comment by jayeshcspl** [comment]
Ok thank you now it's working fine.
Very good package for cdn. 
Thank you very much for you quick support.

**Comment by arubacao** [comment]
No problem. Glad your enjoying the package!

**Comment by jayeshcspl** [comment]
icomoon fonts not loading..

Here is style.css file from where fonts called.

http://bpn.svr.files.public.s3.amazonaws.com/assets/css/icons/icomoon/styles.css

Access to font at 'http://bpn.svr.files.public.s3.amazonaws.com/assets/css/icons/icomoon/fonts/icomoon.woff?3p0rtw' from origin 'http://127.0.0.1:8000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

**Comment by arubacao** [comment]
The error already mentions it: you need to configure cors on your webserver / laravel application. Newer Laravel versions use this out of the box: https://github.com/fruitcake/laravel-cors
